aura.grammar

```diff
@@ -0,0 +1,56 @@
+
+use_statement: identifier ['as' identifier]
+
+function_def: 'fn' NAME '(' decl_list ')' type NEWLINE function_body
```

Comment:
Can you have this? If so, we should use `identifier`. If not, we should use `type`.
```
fn foo.bar ()
```